src/NuGet/Microsoft.NETCore.Compilers.nuspec

```diff
@@ -26,9 +26,12 @@
     <file src="$additionalFilesPath$/Microsoft.NETCore.Compilers.props" target="build" />
 
     <!-- Targets and task files -->
-    <file src="Dlls/MSBuildTask/Microsoft.Build.Tasks.CodeAnalysis.dll" target="tools" />
-    <file src="Dlls/MSbuildTask/Microsoft.CSharp.Core.targets" target="tools" />
-    <file src="Dlls/MSbuildTask/Microsoft.VisualBasic.Core.targets" target="tools" />
+    <file src="Dlls/MSBuildTask/netstandard1.3/publish/Microsoft.Build.Tasks.CodeAnalysis.dll" target="tools" />
+    <file src="Dlls/MSbuildTask/netstandard1.3/publish/Microsoft.CSharp.Core.targets" target="tools" />
+    <file src="Dlls/MSbuildTask/netstandard1.3/publish/Microsoft.VisualBasic.Core.targets" target="tools" />
+    <!-- N.B.: The backslashes below cannot be replaced with forward slashes. -->
+    <file src="Dlls\MSBuildTask\netstandard1.3\publish\System.*.dll" target="tools" />
+    <file src="Dlls\MSBuildTask\netstandard1.3\publish\runtimes\**" target="tools\runtimes" />
```

Comment:
Can we pick `\` or `/` for the slash direction? Don't care which we use, just prefer we use one of them.

Reply:
I'm now worried about changing anything here because I have no idea what it will do to the package layout.